services/src/main/java/org/fao/geonet/api/OpenApiController.java

```diff
@@ -180,9 +189,14 @@ protected boolean isRestController(Map<String, Object> restControllers,
                 || !ModelAndView.class.isAssignableFrom(handlerMethod.getMethod().getReturnType()));
     }
 
-    protected void calculateServerUrl(HttpServletRequest request, String apiDocsUrl) {
```

Comment:
Can you describe a bit the changes related to `calculateServerUrl` / `getServerBaseUrl`

Reply:
Then I noticed that calculateServerUrl was not only calculating a url, it was calculating a url and then setting the serverBaseUrl.   I would have expected that  calculateServerUrl would have returned a value but it was setting a value so I changed it to setServerBaseUrl to make it clearer.
 
I was also working on some other code and I needed to call the functionality from getServerBaseUrl in a couple places so I separated the function to avoid duplicate code. 

I no longer needed to call  getServerBaseUrl in multiple places but thought it was still good to keep it separated.

If you want I can merge the code back into a calculateServerUrl function?